### PR TITLE
fix: Default tab selected issue

### LINKED ISSUE #9490: Wrong tab have been selected
### Issue Summary

If we try to add team member second time. By default it's selected _**invite individual**_ but actually showing _**bulk import**_ screen.

### Steps to Reproduce

1. Choose Teams menu.
2. Click on new or Create Team button.
3. complete step 1. 
4. Click on add team member 
5. choose bulk import.
6. Add or import email and click send invite. 
7. Click on add team member again.

Screenshot 2023-06-13 at 8 42 22 PM

Since it was allowing the user to resume the work where he/she left. A respective tab should be select. Instead it's always selecting a default one, but showing different tab's screen.

### PR DESCRIPTION
**Pull request by VinoV1999**
## What does this PR do?

Fixes the **bulk import** components rendering but **individual invite** tab has been selected issue.

Fixes #9490

https://github.com/calcom/cal.com/assets/123581387/08415c0f-0890-4841-b726-badd2fcae171

## Type of change

- Create a function that reset the values.
- Called that function onSuccess function

Note : The reason to get a function from child to parent and called it in onSuccess function is to avoid clearing the inputs before the service is getting called. That prevents the data if services are down.

### CODE REVIEW

**Review by Udit-takkar — APPROVED** [review]
@VinoV1999 LGTM.  I have simplified the code

**Comment by VinoV1999** [comment]
@Udit-takkar  yes it's looks simple. But that causes below issues.


https://github.com/calcom/cal.com/assets/123581387/a3bd3222-1168-480d-b3a8-0c16c5df367d

Basically resetFields() is getting called even the server throws exception.

### steps to reproduce
https://github.com/calcom/cal.com/blob/main/packages/trpc/server/routers/viewer/teams/_router.tsx

1. Open the above file.
2. At line No. 148 you can find inviteMember method.
3. Paste the below code inside the method.

`if(1==1) throw new Error("Failed to load handler");`

**FYI: This is to interrupt the flow of server.**

File: packages/features/ee/teams/components/AddNewTeamMembers.tsx [hunk 1]
```diff
@@ -121,14 +96,43 @@ export const AddNewTeamMembersForm = ({
...
+                    setMemberInviteModal(false);
+                    if (data.sendEmailInvitation) {
+                      if (Array.isArray(data.usernameOrEmail)) {
+                        showToast(
+                          t("email_invite_team_bulk", {
+                            userCount: data.usernameOrEmail.length,
+                          }),
+                          "success"
+                        );
+                        resetFields();
```
Comment (Udit-takkar) [inline packages/features/ee/teams/components/AddNewTeamMembers.tsx:120]: moved onSuccess here to call resetFields

Comment (Udit-takkar) [reply packages/features/ee/teams/components/AddNewTeamMembers.tsx:120]: Thanks @VinoV1999 this should fix the issue

**Review by sean-brydon — APPROVED** [review]
Nice! i was actually just fixing this locally for another PR :) Great job

File: packages/features/ee/teams/components/MemberInvitationModal.tsx [hunk 2]
```diff
@@ -31,8 +31,8 @@ import { GoogleWorkspaceInviteButton } from "./GoogleWorkspaceInviteButton";
 type MemberInvitationModalProps = {
   isOpen: boolean;
   onExit: () => void;
-  onSubmit: (values: NewMemberForm) => void;
-  onSettingsOpen?: () => void;
+  onSubmit: (values: NewMemberForm, resetFields: () => void) => void;
+  onSettingsOpen: () => void;
```
Comment (emrysal) [inline packages/features/ee/teams/components/MemberInvitationModal.tsx:35]: I don't think this should be required, as it causes type errors

```suggestion
  onSettingsOpen?: () => void;
```

**Review by emrysal — CHANGES_REQUESTED** [review]

**Review by emrysal — APPROVED** [review]

**Pull request merged**